### DISCUSSION TITLE
Issue with Test Label Sending

### DISCUSSION THREAD

**Discussion by mbaggnx** [porrey/Virtual-ZPL-Printer#23] [General]
Good morning, 

Lately I am getting this type of error

Cannot be able to open the test label sending

Can be possible need to be to saturated the cache? or something like that?

<img width="698" alt="image" src="https://user-images.githubusercontent.com/119416732/217201413-b835e83e-1992-47c9-92bd-fd05d4c7b284.png">

**Comment by mbaggnx** [comment]
Also, it does not open again:

<img width="782" alt="image" src="https://user-images.githubusercontent.com/119416732/217227602-96cea87f-a217-42e6-be6a-c83cfbe4968c.png">

**Reply by porrey** [reply]
This issue has been resolved as of version **2.3.0**.

**Reply by mbaggnx** [reply]
> This issue has been resolved as of version **2.3.0**.

Hello!

its happening on 2.3.0

<img width="701" alt="image" src="https://user-images.githubusercontent.com/119416732/217470767-b3dad141-c9ea-40ad-a8ba-c06c9609d95e.png">

**Reply by porrey** [reply]
Please provide more detail. I am unable to reproduce your issue.

**Reply by mbaggnx** [reply]
> Please provide more detail. I am unable to reproduce your issue.

Hello! it happens when you have 2 screens and you left the test screen open in the second screen
Once you unplug the second screen and try to open test sending with your own screen, it shows like this

Thanks, kind regards
